spec.md

```diff
@@ -1,6 +1,7 @@
 # CloudEvents - Verson 0.1
 
-CloudEvents is a vendor-neutral specification for event data.
```

Comment:
note typo above Verson -> Version.

Reply:
Even though I didn't touch that line as part of this PR, I fixed it. thanks!